easybuild/easyconfigs/l/libdeflate/libdeflate-1.15-GCCcore-12.2.0.eb

```
@@ -0,0 +1,36 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+# Updated to 1.8
+# J. Sassmannshausen NHS/GSTT
+
+easyblock = 'CMakeMake'
+
+name = 'libdeflate'
+version = '1.15'
+
+homepage = 'https://github.com/ebiggers/libdeflate'
+description = """Heavily optimized library for DEFLATE/zlib/gzip compression and decompression."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+github_account = 'ebiggers'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['58b95040df7383dc0413defb700d9893c194732474283cc4c8f144b00a68154b']
+
+builddependencies = [('binutils', '2.39')]
```

```suggestion
builddependencies = [
    ('binutils', '2.39'),
    ('CMake', '3.24.3'),
]
```

Sorry. My test system has `CMake` in the OS.